packages/angular_devkit/build_angular/src/angular-cli-files/plugins/scripts-webpack-plugin.ts

```diff
@@ -83,7 +84,9 @@ export class ScriptsWebpackPlugin {
 
     const entrypoint = new EntryPoint(this.options.name);
     entrypoint.pushChunk(chunk);
-
+    if (!this.options.lazy){
+      chunk.addGroup(entrypoint);
```

Comment:
Lazy shouldn’t be a guard for this.   The chunk is already part of the group from the pushChunk call.  The issue here is that it was assumed that the pushChunk call performs the reciprocal action.  The correct behavior is to always execute both actions.

Deciding which entrypoints should be present and how they are represented in an HTML fileis a function of the html processing plugin.

Reply:
In current situation, the chunk [canBeInitial](https://github.com/webpack/webpack/blob/master/lib/Chunk.js#L199) always return false when not put the ChunkGoup in to the chunk. 
The chunk will [filter out](https://github.com/jantimon/html-webpack-plugin/blob/v3.2.0/index.js#L384) even you stat in the filter option.

Reply:
Yes.  The only change that is need here is the addition of `chunk.addGroup(entrypoint);`.  The lazy logic is only relevant to the HTML plugin.

Reply:
ok changed